src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -222,6 +222,14 @@ MUST be present **in this specific order**:
 | type            | REQUIRED              | Type of channel; MUST use the channel types listed below. Note that the type MUST be in upper-case.                                                                           |
 | units           | REQUIRED              | Physical unit of the value represented in this channel, for example, `V` for Volt, or `fT/cm` for femto Tesla per centimeter (see [Units](../02-common-principles.md#units)). |
 
+{{ MACROS___make_columns_table(
+   {
+      "name_channels": "REQUIRED",
```

Comment:
same comment as for MEG above --> there is now ieeg specific info here, which seems out of place.